remoting/zookeeper/client.go

```diff
@@ -513,7 +514,7 @@ func (z *ZookeeperClient) GetChildrenW(path string) ([]string, <-chan zk.Event,
 		return nil, nil, perrors.Errorf("path{%s} has none children", path)
 	}
 	if len(children) == 0 {
-		return nil, nil, perrors.Errorf("path{%s} has none children", path)
+		return nil, nil, errNilChildren
```

Comment:
i think you should print the zkpath as before, otherwise users don't know what is going on.

Reply:
@zouyx , @hxmhlt used the errNilChildren later. so there is no need to add the zkPath in the  error. maybe an error log is better.

Reply:
@AlexStocks ok.